samples/icons-pack/pipeline/registerIcons.ts

```diff
@@ -0,0 +1,45 @@
+import * as fs from "fs";
+import * as path from "path";
+import { fileURLToPath } from "url";
+
+// This script generates a function that invokes registerIcon() for every icon
+// exported by @iconscout/react-unicons
+
+const currentDir = path.dirname(fileURLToPath(import.meta.url));
+const rootDir = path.resolve(currentDir, "../../..");
+const iconsDir = path.join(
+    rootDir,
+    "node_modules/@iconscout/react-unicons/icons"
+);
+
+let registerIcons = `/**
+ * NOTE: THIS CODE IS AUTO-GENERATED. DO NOT MODIFY BY HAND.
+ */
+import type { LibraryRegistry } from "@vertigis/web/config";
+export function registerIcons(registry: LibraryRegistry): void {`;
+
+const files = fs.readdirSync(iconsDir);
+
+for (const file of files) {
+    if (path.extname(file) !== ".js") {
```

Comment:
While being able to consume a library of react icon components is useful, I think there is definitely a need for something that allows people to convert their own .svg files into Web icons, as we do with our icon library.

I think it would be great if this sample allowed people to easily do that. Here is how we could maybe accommodate both:
 - Change this code to also recognize .svg extensions and wrap the inner SVG XML in a `createSvgIcon()` call, exactly as we do.
 - Change the signature to allow an array of icon folders. Include a `svg` folder in the sample with a few .svg files. 
 - Right in the README.md, give instructions to drop their own .svg files in that folder and comment out the path to the unicorn library, OR how to point to a different icon library and comment out the svg folder. The objective would be for someone with little coding experience to successfully turn this sample into a usable production icon library with their own SVG files or an icon library of choice without having to understand everything that is going on in here.